src/i18n/locales/fr.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
This file should also be translated right?

Reply:
Yes but I work part par part